src/goto-analyzer/static_analyzer.cpp

```diff
@@ -8,25 +8,28 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <fstream>
 
-#include <util/threeval.h>
 #include <util/json.h>
 #include <util/xml.h>
 
 #include <analyses/interval_domain.h>
+#include <analyses/constant_propagator.h>
 
 #include "static_analyzer.h"
 
+template<class analyzerT>
```

Comment:
Use `typename` instead of `class` (so as to permitting passing in, e.g., a struct or typedef.